Fourth dance move

deramatd opened the pull request:
-Created dance move as specified in user story #14.
-Added delays in method to account for hardware limitations.
-Added fourth dance move (macarena) to randomiser.

Closes: #14.

deramatd committed:
Create fourth dance move

We created a first attempt at the macarena dance move. We used a while loop and took inspiration
from the other dance move methods we have created. The statements mirror the steps outlined on Github.

Resolves #14

shonaighdouglas committed:
Add delays to fourth dance move

Added 1-second delays between the steps of the dance move to overcome hardware issues

Resolves #14

shonaighdouglas committed:
Add fourth dance move to randomiser

Uncommented the randomiser and added the macarena method to the switch case.

Resolves #14

neginhashmati commented on src/carLogic/carLogic.ino:
```diff
@@ -36,9 +36,8 @@ void loop() {
-      handleInput(danceID);
-
+      int danceID = random(1, 5);
+      handleInput(danceID);*/
```
This */ shouldn't be here. Please remove :)

neginhashmati commented on src/carLogic/carLogic.ino:
```diff
@@ -191,3 +193,43 @@ void shake(int speed) {
+      car.setSpeed(speed);
+      steps++;
+    } else if ((steps == 5) && (car.getDistance() - startingPoint == 0)) {
+     car.setSpeed(0);
```
Please indent this line. :)

neginhashmati commented on src/carLogic/carLogic.ino:
```diff
@@ -191,3 +193,43 @@ void shake(int speed) {
+      repeats++;
+      
+    }
+    /*car.setSpeed(0);
```
Remove all commented out code and all extra lines from line 231 to the last bracket. :)

neginhashmati reviewed (changes requested):
Fix the mentioned issues. Looks good otherwise!

shonaighdouglas committed:
Make corrections suggested in pull request comments

Removed excess space, commented-out code, and fixed tabbing in macarena method.

Resolves #14

neginhashmati reviewed (approved):
Thanks for changing the requested changes, it looks great!
